docs/devcontainer-example/Dockerfile.llvm-project

```
@@ -0,0 +1,21 @@
+# By default, use ubuntu:focal
+ARG BASE_IMAGE="onnxmlirczar/onnx-mlir-llvm-static:amd64"
+FROM ${BASE_IMAGE}
+
+RUN apt-get update
+
+# 2) Install optional packages, comment/uncomment/add as you see fit.
+RUN apt-get install -y vim bash valgrind clang-format lsb-release \
+    wget software-properties-common ssh-client gdb
+RUN python3 -m pip install -U pip && pip install commitizen
+# Install clang
+RUN wget https://apt.llvm.org/llvm.sh && chmod +x llvm.sh \
+    && ./llvm.sh 15 ""
```

Nit: you should remove the script after you're done
```suggestion
    && ./llvm.sh 15 "" && rm llvm.sh
```